README.md

```diff
@@ -23,42 +19,46 @@ Or install it yourself as:
 
     $ gem install rt_rubocop_defaults
 
-## Usage
+## Configuration
 
-You need to tell RuboCop to load the RSpec extension. There are two ways to do this:
+You need to tell RuboCop to load the defaults from this gem. There are two ways
+to do this:
 
 ### RuboCop configuration file
 
-Put this into your .rubocop.yml.
+Put this into your .rubocop.yml:
 
 ```yml
 require: rt_rubocop_defaults
 ```
 
-Now you can run rubocop and it will automatically load the RuboCop RSpec cops together with the standard cops.
+Now you can run rubocop and it will automatically load the RuboCop defaults
+together with the standard cops.
 
 ### Command line
 
 ```sh
 rubocop --require rt_rubocop_defaults
 ```
 
+## Usage
```

Comment:
Isn't the `Configuration` part actually the usage of this gem?

Reply:
yeah I also put it in usage first but then i thought that you actually configure rubocop to then use it with the defaults.... I guess its understandable either way

Reply:
True that.